.github/workflows/windows.yml

```diff
@@ -55,35 +61,47 @@ jobs:
       run: py -3 -m pip install meson
     - name: test meson
       run: meson -v
-    - name: Cache OpenSSL compilation
-      id: cache-openssl
-      uses: actions/cache@v1
-      env:
-        cache-name: cache-openssl-compilation
-      with:
-        path: openssl
-        key: openssl-1_1_1g
-        restore-keys: openssl-1_1_1g
-    - name: Download openssl source
-      if: steps.cache-openssl.outputs.cache-hit != 'true'
-      run: git clone -b OpenSSL_1_1_1g https://github.com/openssl/openssl
     - uses: ilammy/msvc-dev-cmd@v1
       with:
         arch: x64
     - name: Show from where clang-cl
       run: where clang-cl
-    - name: Configure OpenSSL
-      if: steps.cache-openssl.outputs.cache-hit != 'true'
-      run: perl Configure VC-WIN64A
-      working-directory: .\openssl
-    - name: Compile OpenSSL
-      if: steps.cache-openssl.outputs.cache-hit != 'true'
-      run: nmake
-      working-directory: .\openssl
+    - name: Check if vcpkg is already installed
+      if: steps.cache-vcpkg.outputs.cache-hit == 'true'
+      run: .\vcpkg\vcpkg version
+    - name: Clone vcpkg
+      if: steps.cache-vcpkg.outputs.cache-hit != 'true'
+      run: git clone https://github.com/microsoft/vcpkg
+    - name: Checkout to f4bd6423
+      if: steps.cache-vcpkg.outputs.cache-hit != 'true'
+      run: git checkout f4bd6423
+      working-directory: .\vcpkg
+    - name: Run vcpkg bootstrap
+      if: steps.cache-vcpkg.outputs.cache-hit != 'true'
+      run: .\vcpkg\bootstrap-vcpkg.bat
+    - name: Install openssl (vcpkg)
+      if: steps.cache-vcpkg.outputs.cache-hit != 'true'
```

Comment:
Are those `if` still necessary?

Reply:
Yes. We just want to run bootstrap/etc. if the cache hits (you're asking if it shouldn't automatic ignored by the CI?).

Reply:
So shouldn't all package installations with `vcpkg` have that? Or the point is to don't cache `freetype`, `llibjpeg`, `lua`, `zlib`, and `getopt` and only cache `openssl`?

Reply:
Uuuuuuuuuuuuuuuuh yes, I think. Let me see how CI behaves on that.

Reply:
Ok, CI keeps asking vcpkg to install them (although it just ends with a "but it is already installed!"). Now I'm caching everyone.